.eslintrc.json

```diff
@@ -11,5 +11,7 @@
   "parserOptions": {
     "ecmaVersion": 12
   },
-  "rules": {}
+  "rules": {
+    "no-console": "off"
```

Comment:
Why did we add this rule?

Reply:
Linter at one point was throwing me a warning to not include console.logs. Since the code base was using console.logs to output errors, I toggled off that setting. However, I removed that rule, tested again locally, and am no longer seeing the warning.

Reply:
I'd err just to keep the warning (meaning, don't turn the rule off) since it doesn't prevent compilation of the app.  It's also out of scope for the PR.